static/src/javascripts/projects/common/modules/commercial/contributions-service.js

```diff
@@ -245,6 +266,21 @@ const getPuzzlesBanner = (payload) => {
     });
 }
 
+const getHeaderLinks = (payload) => {
+    const isProd = config.get('page.isProd');
+    const url = isProd ? 'https://contributions.guardianapis.com/header' : 'https://contributions.code.dev-guardianapis.com/header';
+    const json = JSON.stringify(payload);
+
+    const forcedVariant = getForcedVariant('header');
+    const queryString = forcedVariant ? `?force=${forcedVariant}` : '';
+
+    return fetchJson(`${url}${queryString}`, {
+        method: 'post',
```

Comment:
Method verbs should be uppercase.

```suggestion
        method: 'POST',
```